examples/sensor-data-scala/README.md

```diff
@@ -121,14 +122,37 @@ spec:
       storage: 1Gi
 ```
 
+claim-eks.yaml:
+
+```
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: claim1
+  namespace: sensor-data-scala
+spec:
+  accessModes:
+    - ReadWriteMany
+  storageClassName: gp2
```

Comment:
@eschizoid are you sure this works? AFAIK the `gp2` storage class only supports `RWO` access mode.

Reply:
Ah, the template version does specify RWO so this text just needs to be adjusted to match.

Reply:
>@eschizoid are you sure this works? AFAIK the gp2 storage class only supports RWO access mode.
You are right this is not working. See my previous comment, we don't have support RWM on AWS

I tried setting it to `ReadWriteOnce` and it didn't, i think helm chars also need to be adjusted?

Reply:
Well, the installer will ask you to explicitly choose a storage class that supports RWM for the Spark/Flink checkpointing volumes. It leaves this to the user since it doesn't want to enforce a specific choice and it can't inspect the cluster with absolute knowledge about which storage classes support RWM since unfortunately K*s doesn't have an API for that 😢 

What we _can_ do is to test with EFS, which  is something we have on an internal backlog as well if you don't have time for that. And then we can update the docs to add a not that for EKS the recommended RWM storage class is EFS (plus any necessary installation/configuration notes).

@eschizoid Could you perhaps help test with EFS? 
If not, we will definitely still merge this PR but then we will have to schedule some time to do the testing ourselves and that might take a little longer.

Reply:
@agemooij I would be happy to do it, but maybe is better to just start with plain NFS for both EKS and GKE? meaning we install the NFS server for GKE and EKS. If that doesn't work we can move the the specialized version and used AWS EFS with EKS.

Thoughts?